composer.json

```diff
@@ -10,7 +10,7 @@
     ],
     "require": {
         "php": "^5.3.3|^7.0",
-        "justinrainbow/json-schema": "^2.0",
+        "justinrainbow/json-schema": "^3.0",
```

Comment:
thanks for the hint @staabm! I think it would indeed be good to support all three branches. @thewilkybarkid could you adapt the PR?